src/commands/utils/avatar.ts

```diff
@@ -7,12 +7,21 @@ export const avatar: CommandDefinition = {
     description: 'Shows the selected user\'s avatar',
     category: CommandCategory.UTILS,
     executor: async (msg) => {
-        const user = msg.mentions.users.first() || msg.author;
+        const split = msg.content.replace(/(?:\.av|\.avatar)\s+/, '').split(' ');
+        const userID = msg.guild.members.cache.get(split[0])
+        const user = msg.mentions.users.first() || userID || msg.author;
         user.displayAvatarURL({ dynamic: true });
         const avatarEmbed = makeEmbed({
-            title: `${user.tag}'s Avatar`,
+            title: '',
```

Comment:
You can remove this line as you are setting a title later within your `if` statements